dot/config.go

```diff
@@ -252,38 +254,43 @@ func WestendDevConfig() *Config {
 }
 
 // KusamaConfig returns a kusama node configuration
-func KusamaConfig() *Config {
+func KusamaConfig() (*Config, error) {
+	defaultLogLevel, err := log.ParseLevel(westend.DefaultLvl)
```

Comment:
should we pass in kusama default here instead of westend?

Reply:
done